src/vault/env.clj

```diff
@@ -22,13 +22,24 @@
   "A Vault authentication token which should be used directly by the client."
   :secret true)
 
+(defenv :wrap-vault-token
```

Comment:
This should begin with `vault-` to follow the prefixes on the other variables. Perhaps `:vault-wrap-token`?